lib/rubocop/cop/style/redundant_line_continuation.rb

```diff
@@ -155,8 +158,9 @@ def inside_string_literal?(range, token)
         #
         #   do_something \
         #     argument
-        def method_with_argument?(current_token, next_token)
+        def method_with_argument?(line_range, current_token, next_token)
           return false unless ARGUMENT_TAKING_FLOW_TOKEN_TYPES.include?(current_token.type)
+          return false unless current_token.pos.overlaps?(line_range)
```

Comment:
Previously `method_with_argument?` was not considering the range, but rather the full set of tokens in the source. This meant if there was a pair of tokens where `method_with_argument?` would be true, all following continuations would be accepted.